app/templates/index.html

```diff
@@ -46,6 +48,205 @@ <h4>LET’S CONNECT</h4>
                 </div>
             </div>
         </section>
+
+        <!--Middle section one-->
+        <section class="section-elements">
+            <div class="middle-section">
+                <div class="section-one-bg"></div>
+                <div class="section-block">
+                    <div class="section-title"><h2>Tonic</h2></div>
+                    <div class="section-frame">
+                        <div class="cannopy">CANOPY
+                            <div class="section-img">
+                                <img src="/app/static/images/Counter.svg" alt="">
+                            </div>
+                        </div>
+                        <div class="backend">Back End Dev
+                            <div class="section-img">
+                                <img src="/app/static/images/Counter.svg" alt="">
+                            </div> 
+                        </div>
+                        <div class="year">2015</div>
+                    </div>
+                    <div class="section-primary-text">
+                        <p>
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="section-tags">
+                        <button type="submit">html</button>
+                        <button type="submit">css</button>
+                        <button type="submit">javascript</button>
+                    </div>
+                    <div class="section-action">
+                        <button type="submit">See project</button>
+                    </div>
```

Comment:
- [ ] You have done a great job styling your button. There is something very small remaining. Kindly implement the button states eg button on hover, button clicked etc. Currently your buttons are static and no effect is seen when you click on them. 

**From the Figma design, the following are the style guides for the button states:**

![image](https://user-images.githubusercontent.com/86318284/185554481-e54624d3-522d-424e-ac9a-36f9fc2266f6.png)

If you might not be knowing how to complement this, have a look at [this wonderful resource](https://zellwk.com/blog/style-hover-focus-active-states/).

**Kindly implement this on all the buttons in the project as I have realized that none has this implemented.**